src/Location/Location.test.tsx

```diff
@@ -1,35 +1,237 @@
 import React from "react";
-import { render, getAllByRole} from "@testing-library/react";
+import { render, getAllByRole, getAllByText, getByAltText} from "@testing-library/react";
 import Location from './Location'
+import { MemoryRouter } from "react-router-dom";
+import { mocked } from "ts-jest/utils";
+jest.mock('../helpers/apiCalls.tsx')
 
 describe('Location', () => {
+  // not sure how to test for both the fetched data and the weekly
```

Comment:
The full mocked result includes both current and weekly as a single object with two keys (currentWeather and weeklyWeather) so you will just mock the full object one time any time you are testing for current or for weekly.